CONTRIBUTING.md

```diff
@@ -46,3 +46,13 @@ The outline above is an extremely simplified version. In real life you have to d
 - Make sure test-suite passes: `npm test`
 - Squash your commits
 - Write a convincing description of your PR and why we should land it
+
+## Release Checklist
+
+- Run [`npm version patch`](https://docs.npmjs.com/cli/version) to update the version number. Use `minor` or `major` instead of `patch` if needed (see [semver.org](http://semver.org/) for details).
+- Open a [pull request](https://github.com/OctoLinker/browser-extension/pulls) with the new version.
+- Once the pull request is merged in, tag the resulting commit as `vX.Y.Z` (where `X`, `Y`, `Z` are the major, minor, and patch versions).
+- Push the tag to GitHub. This will trigger Travis CI to create a new [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) and submit the new Chrome extension to the Chrome Web Store. See [.travis.yml](https://github.com/OctoLinker/browser-extension/blob/master/.travis.yml) for details.
+- Submit `firefox-octolinker-X.Y.Z.zip` from the [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) to [addons.mozilla.org](https://addons.mozilla.org/en-US/developers/addon/octolinker/versions#version-upload). Be sure to include the `Source code (zip)` file from the release as well.
+- Submit `opera-octolinker-X.Y.Z.zip` from the [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) to [addons.opera.com](https://addons.opera.com/developer/package/226344/?tab=versions). Afterwards, go to the [Conversation tab](https://addons.opera.com/developer/package/226344/?tab=conversation), add a link to the `Source code (zip)` file and copy/paste the build instructions from previous releases.
+- Update release notes at https://github.com/OctoLinker/browser-extension/releases/tag/vX.Y.Z
```

Comment:
It sounds like [Travis will simply edit a pre-existing release for the tag, if the release is there](https://github.com/travis-ci/dpl/issues/155#issuecomment-49194900). However, the release cannot be created until the tag is pushed. But Travis runs as soon as the tag is pushed, so there may not be enough time to draft the release notes before Travis adds the zip files and publishes the release.

I probably won't look into adding changelog/release-notes generation to Travis in the near future, especially since there are other parts of the release process that are also manual (in a perfect world, everything would be automated, so we could release as often as we like with no extra effort 😛). Maybe later, if I get the urge to.